src/views/Subscription.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="text-gray-700 bakery-bg">
```

Comment:
This is the subscription view. Eventually I may want to include the items that are tied to the subscription and give the user the ability to edit what items are included in the shipments